### PR TITLE
value in Pad operator should be float

### PR DESCRIPTION
**Pull request by ir5**
This will solve #121.

### CODE REVIEW

File: tests/functions_tests/test_arrays.py [hunk 1]
```diff
@@ -37,6 +37,12 @@
      'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
               'mode': 'edge'},
      'name': 'pad_edge'},
+    {'ops': 'pad', 'input_shape': (1, 2, 3, 4),
+     'input_argname': 'x',
+     'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
+              'mode': 'constant',
+              'constant_values': -1},
```
Comment (shinh) [inline tests/functions_tests/test_arrays.py]: Not your fault, but could you test a random float (e.g., `constant_values=4.2`) as well?

File: tests/functions_tests/test_arrays.py [hunk 2]
```diff
@@ -37,6 +37,12 @@
      'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
               'mode': 'edge'},
      'name': 'pad_edge'},
+    {'ops': 'pad', 'input_shape': (1, 2, 3, 4),
+     'input_argname': 'x',
+     'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
```
Comment (shinh) [inline tests/functions_tests/test_arrays.py]: Not your fault, but could you try using a more varieties of `pad_width`? For example, it's unfortunate there is no imbalanced pads (e.g., `(1, 3)`)

File: onnx_chainer/functions/array.py [hunk 3]
```diff
@@ -130,7 +130,9 @@ def convert_Pad(func, opset_version, input_names, num_outputs,
                 'ONNX doesn\'t support multiple constant values for Pad '
                 'operation')
         elif not isinstance(values, int):
-            values = values[0]
+            values = float(values[0])
```
Comment (shinh) [inline onnx_chainer/functions/array.py]: Does line 128 work for `float` or `np.array(3)` constant_values?

File: tests/functions_tests/test_arrays.py (hunk 1 above)
Comment (ir5) [reply tests/functions_tests/test_arrays.py]: It seems like `chainer.functions.pad` accepts only integer type for `constant_values`.

File: tests/functions_tests/test_arrays.py (hunk 2 above)
Comment (ir5) [reply tests/functions_tests/test_arrays.py]: Added some test case.

**Pull request merged**